pkg/securitypolicy/mounts.go

```diff
@@ -0,0 +1,91 @@
+//go:build linux
+// +build linux
+
+package securitypolicy
+
+import (
+	"github.com/opencontainers/runtime-spec/specs-go"
+
+	specInternal "github.com/Microsoft/hcsshim/internal/guest/spec"
+)
+
+// getDefaultMountConstraints generates mount constraints for default mounts
+// injected by containerD and/or GCS.
+func getDefaultMountConstraints(sandboxID string, spec *specs.Spec) ([]mountInternal, error) {
```

Comment:
This never returns an error, and doesn't seem to be part of implementing an interface, so can remove that from the signature.

Reply:
good point, done.